Jellyfin.Api/Controllers/LiveTvController.cs

```diff
@@ -0,0 +1,1238 @@
+        [SuppressMessage("Microsoft.Performance", "CA1801:ReviewUnusedParameters", MessageId = "imageTypeLimit", Justification = "Imported from ServiceStack")]
+        [SuppressMessage("Microsoft.Performance", "CA1801:ReviewUnusedParameters", MessageId = "enableImageTypes", Justification = "Imported from ServiceStack")]
+        [SuppressMessage("Microsoft.Performance", "CA1801:ReviewUnusedParameters", MessageId = "fields", Justification = "Imported from ServiceStack")]
+        [SuppressMessage("Microsoft.Performance", "CA1801:ReviewUnusedParameters", MessageId = "enableUserData", Justification = "Imported from ServiceStack")]
+        [SuppressMessage("Microsoft.Performance", "CA1801:ReviewUnusedParameters", MessageId = "enableTotalRecordCount", Justification = "Imported from ServiceStack")]
+        public ActionResult<QueryResult<BaseItemDto>> GetRecordingsSeries(
+            [FromQuery] string? channelId,
+            [FromQuery] Guid? userId,
+            [FromQuery] string? groupId,
+            [FromQuery] int? startIndex,
+            [FromQuery] int? limit,
+            [FromQuery] RecordingStatus? status,
+            [FromQuery] bool? isInProgress,
+            [FromQuery] string? seriesTimerId,
+            [FromQuery] bool? enableImages,
+            [FromQuery] int? imageTypeLimit,
+            [FromQuery] string? enableImageTypes,
+            [FromQuery] string? fields,
+            [FromQuery] bool? enableUserData,
+            [FromQuery] bool enableTotalRecordCount = true)
+        {
+            return new QueryResult<BaseItemDto>();
+        }
+
+        /// <summary>
+        /// Gets live tv recording groups.
+        /// </summary>
+        /// <param name="userId">Optional. Filter by user and attach user data.</param>
+        /// <response code="200">Recording groups returned.</response>
+        /// <returns>An <see cref="OkResult"/> containing the recording groups.</returns>
+        [HttpGet("Recordings/Groups")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [Authorize(Policy = Policies.DefaultAuthorization)]
+        [Obsolete("This endpoint is obsolete.")]
+        [SuppressMessage("Microsoft.Performance", "CA1801:ReviewUnusedParameters", MessageId = "userId", Justification = "Imported from ServiceStack")]
+        public ActionResult<QueryResult<BaseItemDto>> GetRecordingGroups([FromQuery] Guid? userId)
+        {
+            return new QueryResult<BaseItemDto>();
+        }
+
+        /// <summary>
+        /// Gets recording folders.
+        /// </summary>
+        /// <param name="userId">Optional. Filter by user and attach user data.</param>
+        /// <response code="200">Recording folders returned.</response>
+        /// <returns>An <see cref="OkResult"/> containing the recording folders.</returns>
+        [HttpGet("Recordings/Folders")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [Authorize(Policy = Policies.DefaultAuthorization)]
```

Comment:
It's a little inconsistent whether DefaultAuth is on every endpoint or at Controller level.

Reply:
auth is on the controller level if every endpoint has auth, otherwise it's at the endpoint level.